src/components/DownloadLoginFile.js

```diff
@@ -104,12 +107,12 @@ class DownloadLoginFile extends Nimiq.Observable {
     _setupDownload(href, filename) {
         if (this._supportsNativeDownload()) {
             // Setup native download
-            this.$el.href = href;
-            this.$el.download = filename;
+            this.$downloadButton.href = href;
+            this.$downloadButton.download = filename;
         } else {
             // Setup fallback download
             // Hack to make image downloadable on iOS via long tap.
-            this.$el.href = 'javascript:void(0);'; // eslint-disable-line no-script-url
+            // this.$downloadButton.href = 'javascript:void(0);'; // eslint-disable-line no-script-url
```

Comment:
no longer needed?

Reply:
It was. Haven't had time or the iPhone to test it before. Fixed it.